airframe-http-okhttp/src/main/scala/wvlet/airframe/http/okhttp/OkHttpClient.scala

```diff
@@ -242,7 +242,10 @@ class OkHttpClient(address: ServerAddress, config: OkHttpClientConfig)
     convert[OperationResponse](send(r, requestFilter))
   }
 
-  def close(): Unit = {}
+  def close(): Unit = {
```

Comment:
ok. Then, let's implement AutoCloseable and override close() method

Reply:
`HttpSyncClient` already has `AutoCloseable`, but we hasn't gave `override` modifier to `close()` method!